### PR TITLE
AOS-2445 always add the basic secretVault handler

### PR DESCRIPTION
**Pull request by bjartek**
When doing exteneded/simple config we need to remember that some files can have simple config and others advanced. So we always need to push all the handlers.

### CODE REVIEW

**Comment by bjartek** [comment]
En full reformat av all groovy og kotlin kode som egen PR hadde egentlig gjort seg.

File: src/test/groovy/no/skatteetaten/aurora/boober/service/AuroraDeploymentSpecValidatorTest.groovy [hunk 1]
```diff
@@ -14,159 +13,171 @@ import no.skatteetaten.aurora.boober.service.vault.VaultService
...
+        def vaultCollection = deploymentSpec.environment.affiliation
+
+        vaultService.vaultExists(vaultCollection, "test2") >> true
+        vaultService.vaultExists(vaultCollection, "test") >> true
+
+        expect:
+        specValidator.validateVaultExistence(deploymentSpec)
+    }
+
+    def "should handle overrides of secretVaultKeuys with different syntax"() {
```
Comment (laukus) [inline src/test/groovy/no/skatteetaten/aurora/boober/service/AuroraDeploymentSpecValidatorTest.groovy]: secretVaultKeuys

File: src/main/kotlin/no/skatteetaten/aurora/boober/mapper/v1/AuroraVolumeMapperV1.kt [hunk 2]
```diff
@@ -28,20 +28,11 @@ class AuroraVolumeMapperV1(private val applicationFiles: List<AuroraConfigFile>)
...
-                    AuroraConfigFieldHandler("secretVault")
-            )
-        }
-
-        return secretVaultSubKeys.flatMap { secretVaultSubKey ->
-            listOf(
-                    AuroraConfigFieldHandler("secretVault/$secretVaultSubKey")
-            )
-        }
+        return listOf(
```
Comment (bjartek) [inline src/main/kotlin/no/skatteetaten/aurora/boober/mapper/v1/AuroraVolumeMapperV1.kt]: I am not sure why I let this slip in the original PR. Sorry.

Comment (laukus) [reply src/main/kotlin/no/skatteetaten/aurora/boober/mapper/v1/AuroraVolumeMapperV1.kt]: not much of a slip, but it is clearer now

**Pull request merged**